app/templates/partials/widgets/multiple_choice_widget.html

```diff
@@ -23,7 +23,7 @@
       "name": option.name,
       "id": option.id,
       "checked": "checked" if option['checked'],
-      "data-qa": "has-other-option" if answer.id in form.other_answer and form.other_answer[answer.id][loop.index0] != None
+      "data-qa": "has-other-option" if option.detail_answer_id
```

Comment:
Depends how picky we are being. "has-other-option" or "has-detail-answer"?

Reply:
👍 Not used so deleted.